doc/mgr/orchestrator.rst

```diff
@@ -180,9 +183,25 @@ Example::
     all-available-devices node2 /dev/vdc -  -
     all-available-devices node3 /dev/vdd -  -
 
+When it is used the parameter ``all-available-devices``, a cephadm service is created.
+This service guarantees that all the available devices in all the cluster nodes will be used for OSD's.
+You should take into account the implications of this automatic behaviour enabled by default.
```

Comment:
This is another somewhat nitpicky comment, but I think I'd rewrite this sentence:
You should take into account the implications of this automatic behaviour enabled by default.

s/ You should take into account the implications of this automatic behaviour enabled by default./Take into account the implications of this behavior, which is automatic and enabled by default./